docs/quickstart.html

```diff
@@ -121,7 +121,7 @@ <h4 class="anchor-heading">
             By default, each line you enter will result in a separate event being written to the topic.
         </p>
 
-        <pre class="line-numbers"><code class="language-bash">$ bin/kafka-console-producer.sh --topic quickstart-events --bootstrap-server localhost:9092
+        <pre class="line-numbers"><code class="language-bash">&gt; bin/kafka-console-producer.sh --topic quickstart-events --bootstrap-server localhost:9092
```

Comment:
The console producer tool prints a greater-than character for each new line, so the lines below should be:
```
>This is my first event
>This is my second event</code></pre>
````